.stylelintrc

```diff
@@ -64,7 +64,7 @@
     "selector-pseudo-class-case": "lower",
     "selector-pseudo-element-case": "lower",
     "selector-type-case": "lower",
-    "selector-type-no-unknown": true,
+    "selector-type-no-unknown": [true, { "ignoreTypes": ["/^_/","/^x/"] }],
```

Comment:
This change seems unrelated to this PR, since it doesn't contain any css code changes.

Reply:
I think Mehul added it back when he had the CSS as part of the commits, but I've removed it.